code/game/dna/dna2.dm

```diff
@@ -189,6 +190,40 @@ GLOBAL_LIST_EMPTY(bad_blocks)
 		val=rand(1, 2049)
 	SetUIValue(block, val, defer)
 
+//Get Tri State Block State
+/datum/dna/proc/GetUITriState(block)
+	if(block <= 0)
+		return
+	else
+		var/val = GetUIValue(block)
+		if(val >= 1395 && val <= 2760) //female
+			return 0
+		if(val >= 1 && val <= 1395) //male
```

Comment:
the comment makes no sense here since this is a generic helper

Reply:
Oh! I put that in there because this will change how the values are assigned and we'll need to update the Wiki with the new Hexcodes should this get merged:
https://www.paradisestation.org/wiki/index.php/Guide_to_Genetics#Unique_Identifiers_.28UI.29

Instead of: 
Gender:
< 800 = Male
> 800 = Female

It will be:
Gender:
< 54F = Male
> 54F = Female = < 81C
> 81C Plural